### ISSUE TITLE
As a developer I want to be able to use blake2b hashing functionality

### ISSUE THREAD

**Issue by nshCore** [atlascity/net-ipfs-engine#3]
To match our codebase we need to use the blake2b hashing funcitonality

**Comment by richardschneider** [comment]
`blake2b` is already supported, see [hashing algorithms](https://richardschneider.github.io/net-ipfs-core/articles/multihash.html#algorithms).

**Comment by nshCore** [comment]
hi @richardschneider didn't see this, will speak more about how to integrate this, i'm presuming it uses cs-multihash

**Comment by richardschneider** [comment]
No, `net-ipfs-core` is using Bouncy Castle for blake2b, see the [hash algorithm registry](https://github.com/richardschneider/net-ipfs-core/blob/8d1343839ad6ec4ddd5849f467d64af26742440b/src/Registry/HashingAlgorithm.cs#L65).
